unified-codes/release.sh

```diff
@@ -0,0 +1,30 @@
+### unified codes release steps                     ###
+### before using, add your public key to the server ###
+### to allow scp & ssh                              ###
+
+################ web ################
+ cp .dist.env .env
+ mv .local.env .local.env.backup
+
+ nx build web --prod --optimization
+ rm .env
```

Comment:
Nit: if you want `rm .env` only to work on success, can do: `nx build web --prod --optimization && rm .env`.